### PR TITLE
Adding assessments and an explanation of changes coming in v2 APIs

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: _articles/assessments.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+The [Assessments API](/swagger-ui/index.html#/Assessments) has two main components. First, an [Assessment](/model-browser.html#Assessment) and its associated [ExternalResource](/model-browser.html#ExternalResource) links provide documentation for the assessment (both technical and scientific). We expect most documentation will live outside of our shared assessment tools, hosted in places like Sage Bionetwork’s [Synapse wikis](https://docs.synapse.org/articles/wikis.html) or on [Github](https://github.com/). Second, each assessment has an associated [configuration] that Bridge’s client app SDK uses to present the user interface in an app.
```
Comment (DanWebster) [inline _articles/assessments.md]:
```suggestion
The [Assessments API](/swagger-ui/index.html#/Assessments) has two main components. First, an [Assessment](/model-browser.html#Assessment) and its associated [ExternalResource](/model-browser.html#ExternalResource) links provide documentation for the assessment (both technical and scientific). We expect most documentation will live outside of our shared assessment tools, hosted in places like Sage Bionetworks' [Synapse wikis](https://docs.synapse.org/articles/wikis.html) or on [Github](https://github.com/). Second, each assessment has an associated [configuration] that Bridge’s client app SDK uses to present the user interface in an app. 
```

Comment (DanWebster) [reply _articles/assessments.md]: I'm assuming here that [configuration] will get a hyperlink when its ready

File: _articles/assessments.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+|license|The software or other licenses for this assessment.|
+|publication|A publication that was based at least in part on data collected through this assessment.|
+|release_note|Documentation of changes in a specific revision of an assessment (we suggest that the `minRevision` and `maxRevision` values on this `ExternalResource` should both be set to the revision being documented).|
+|sample_app|A sample app showing the implementation of the assessment in working software. Intended for developers who are implementing a new app.|
+|sample_data|Sample data for researchers  that is identical to the data generated by the assessment.|
+|screenshot|An image file (`format` field should be an image MIME type) showing an example of the assessment’s UI within Bridge’s app SDKs.|
+|see_also|Link to a related assessment, measure, or methodology (it does not need to be in the Bridge system).|
+|used\_in\_study|A link to a study that has utilized this assessment (this is a link to some public face for the study, not data or publications from that study).|
+|website|Any other website.|
```
Comment (DanWebster) [inline _articles/assessments.md:41]: Might be useful to suggest that if someone wants to include a video preview of their task, that they could upload to youtube as a public video and include in this field

File: _articles/assessments.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+|other|Any other kind of resource that might not be linkable (if you  can link to it, it may be better to categorize it as a website).|
+
+To use a shared assessment, it must be [imported into you app](/swagger-ui/index.html#/Shared%20Assessments/importSharedAssessment). You must be signed in to the app context on Bridge, and you must include the ID of the organization that will own the local copy of the shared assessment (in most cases, there is only one choice here: your own organization). Everyone working on an app (in all organizations) will be able to see and schedule this assessment, but only members of the owning organization will be able to edit or customize the assessment.
```
Comment (DanWebster) [inline _articles/assessments.md]:
```suggestion
To use a shared assessment, it must be [imported into your app](/swagger-ui/index.html#/Shared%20Assessments/importSharedAssessment). You must be signed in to the app context on Bridge, and you must include the ID of the organization that will own the local copy of the shared assessment (in most cases, there is only one choice here: your own organization). Everyone working on an app (in all organizations) will be able to see and schedule this assessment, but only members of the owning organization will be able to edit or customize the assessment.
```

**Review by DanWebster — CHANGES_REQUESTED** [review]
I put in a suggested commit to fix 2 typos that can be directly committed through GitHub's inline commit feature, and the rest are just suggestions.  Classifying as 'request changes' for the typos but otherwise looks great.

File: _articles/assessments.md (hunk 2 above)
Comment (alxdarksage) [reply _articles/assessments.md:41]: Should we include a video_preview category?

**Review by DanWebster — APPROVED** [review]

**Pull request merged**